rke/k8s_version_info.go

```diff
@@ -88,6 +88,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.3.7",
 			MinRKEVersion:     "1.0.7",
 		},
+		"v1.15.12-rancher1-1": {
+			MinRancherVersion: "2.3.3",
```

Comment:
Can this be 2.3.7 to match the latest 1.15.X in 2.3?

Reply:
What about users who upgrade from 2.2.13 to any version <2.3.7?

Reply:
Adding version info was required because of validation checks, I've arranged the list to add an exception for 1.15.12-rancher1-1